lib/purchases_flutter.dart

```diff
@@ -584,6 +591,15 @@ enum PurchaseType {
   subs
 }
 
+/// Billing Feature types
+enum BillingFeature {
+  subscriptions,
+  subscriptions_update,
+  in_app_items_on_vr,
+  subscriptions_on_vr,
+  price_change_confirmation
```

Comment:
convention [for enum values is lowerCamelCase](https://dart.dev/guides/language/effective-dart/style#prefer-using-lowercamelcase-for-constant-names)